scipy/linalg/_basic.py

```diff
@@ -624,12 +624,20 @@ def solve_toeplitz(c_or_cr, b, check_finite=True):
         Whether to check that the input matrices contain only finite numbers.
         Disabling may give a performance gain, but may result in problems
         (result entirely NaNs) if the inputs do contain infinities or NaNs.
+    return_reflection : bool, optional
+        Whether to return the Toeplitz reflection coefficients.
 
     Returns
     -------
     x : (M,) or (M, K) ndarray
         The solution to the system ``T x = b``. Shape of return matches shape
         of `b`.
+    reflection_coeff : (M+1,) or (M+1, K) ndarray
+        Toeplitz reflection coefficients. When the matrix formed by c is
+        symmetric Toeplitz and ``b`` is ``c[n:]``, as in the solution of
```

Comment:
```suggestion
        symmetric Toeplitz and `b` is ``c[n:]``, as in the solution of
```

Reply:
What is the difference? Single backticks for actual symbols and double for other expressions?

Reply:
Exactly, single for variables etc see https://numpydoc.readthedocs.io/en/latest/format.html#common-rest-concepts